src/utils.ts

```diff
@@ -1,6 +1,7 @@
 import https from 'https'
 import http from 'http'
 import { URL } from 'url'
+import { News } from './config'
```

Comment:
```suggestion

import { News } from './config'
```

- should append one line.